assets/scss/base/form/_label.scss

```diff
@@ -275,6 +269,14 @@ Styleguide Label.block
     @extend %vertically-aligned-input;
   }
 
+  // to align radio button with top line of content, not centred
+  input.top-aligned-input {
```

Comment:
Yes, good point. I've now moved `input.top-aligned-input` out of `.block-label`.